theano/gpuarray/tests/test_elemwise.py

```diff
@@ -197,7 +197,7 @@ def test_perform_nan(self):
 
     def setUp(self):
         super(test_GpuCAReduceCuda, self).setUp()
-        if get_context(test_ctx_name).kind != 'cuda':
+        if get_context(test_ctx_name).kind != b'cuda':
```

Comment:
why did you put the kind as byte? It seem to make using it ugly from Theano point of view.